lib/Command/ExportAccount.php

```diff
@@ -34,22 +36,58 @@ protected function configure() {
 		$this->setName('mail:account:export');
 		$this->setDescription('Exports a user\'s IMAP account(s)');
 		$this->addArgument(self::ARGUMENT_USER_ID, InputArgument::REQUIRED);
+		$this->addOption(self::ARGUMENT_OUTPUT_FORMAT, '', InputOption::VALUE_OPTIONAL);
 	}
 
 	protected function execute(InputInterface $input, OutputInterface $output): int {
 		$userId = $input->getArgument(self::ARGUMENT_USER_ID);
 
 		$accounts = $this->accountService->findByUserId($userId);
 
-		foreach ($accounts as $account) {
-			$output->writeln("<info>Account " . $account->getId() . ":</info>");
-			$output->writeln("- E-Mail: " . $account->getEmail());
-			$output->writeln("- Name: " . $account->getName());
-			$output->writeln("- Provision: " . ($account->getMailAccount()->getProvisioningId() ? "set" : "none"). " ID: " . ($account->getMailAccount()->getProvisioningId() ? $account->getMailAccount()->getProvisioningId():"N/A"));
-			$output->writeln("- IMAP user: " . $account->getMailAccount()->getInboundUser());
-			$output->writeln("- IMAP host: " . $account->getMailAccount()->getInboundHost() . ":" . $account->getMailAccount()->getInboundPort() . ", security: " . $account->getMailAccount()->getInboundSslMode());
-			$output->writeln("- SMTP user: " . $account->getMailAccount()->getOutboundUser());
-			$output->writeln("- SMTP host: " . $account->getMailAccount()->getOutboundHost() . ":" . $account->getMailAccount()->getOutboundPort() . ", security: " . $account->getMailAccount()->getOutboundSslMode());
+		function getAccountsData($accounts) {
+			$accountsData = [];
+		
+			foreach ($accounts as $account) {
+				$accountsData[] = [
+					'id' => $account->getId(),
+					'email' => $account->getEmail(),
+					'name' => $account->getName(),
+					'provision' => [
+						'status' => $account->getMailAccount()->getProvisioningId() ? "set" : "none",
+						'id' => $account->getMailAccount()->getProvisioningId() ?: "N/A"
+					],
+					'imap' => [
+						'user' => $account->getMailAccount()->getInboundUser(),
+						'host' => $account->getMailAccount()->getInboundHost(),
+						'port' => $account->getMailAccount()->getInboundPort(),
+						'security' => $account->getMailAccount()->getInboundSslMode()
+					],
+					'smtp' => [
+						'user' => $account->getMailAccount()->getOutboundUser(),
+						'host' => $account->getMailAccount()->getOutboundHost(),
+						'port' => $account->getMailAccount()->getOutboundPort(),
+						'security' => $account->getMailAccount()->getOutboundSslMode()
+					]
+				];
+			}
+		
+			return $accountsData;
+		}
+		if ($input->getOption(self::ARGUMENT_OUTPUT_FORMAT) == 'json') {
```

Comment:
```suggestion
		if ($input->getOption(self::ARGUMENT_OUTPUT_FORMAT) === 'json') {
```

same for the other comparisons. we always use strict comparison in nextcloud. it will also make sure that we are comparing two strings :)